Python/codecs.c

```diff
@@ -618,6 +639,20 @@ int PyCodec_RegisterError(const char *name, PyObject *error)
                                 name, error);
 }
 
+int PyCodec_UnregisterError(const char *name)
+{
+    for (size_t i = 0; i < CODECS_ERROR_POLICY_COUNT; ++i) {
```

Comment:
You can use Py_ARRAY_LENGTH() instead of CODECS_ERROR_POLICY_COUNT.

Reply:
Funny that I actually suggested this in another PR but forgot about it in this one :') Will do, thanks!